3.6/aql/fundamentals.md

```diff
@@ -14,5 +14,5 @@ AQL Fundamentals
 * [Query Results](fundamentals-query-results.html): the result of an AQL query is an array of values. 
 * [Query Errors](fundamentals-query-errors.html): errors may arise from the AQL parsing or execution. 
 
-Learn more about optimizing your queries by going through the
-[Performance Course](https://www.arangodb.com/arangodb-performance-course/){:target="_blank"}.
+You can learn more about optimizing your queries by going through the
```

Comment:
This is about the very basics, so it suggests to optimize queries which were not even written?!

Reply:
ok, let's remove

Reply:
Moved Performance Course & Oasis ad to aql\execution-and-performance.md